apps/schools/migrations/0023_auto_20170630_0642.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
The following sql statements need to be run on the db either via a script or manually to make sure the ENUMs are updated to match what the data is there in the sheet given to us by the Odisha Govt.